src/components/Atoms/Badge/Badge.story.vue

```diff
@@ -0,0 +1,48 @@
+<script lang="ts" setup>
+import { reactive } from 'vue';
+import Badge from './Badge.vue';
+
+import type { BadgeColors, BadgeProps, BadgeVariants } from './types';
+
+const state = reactive<BadgeProps>({
+  text: 'Badge',
+  variant: 'status',
+  size: 'medium',
+  disabled: false
+});
+
+const colorOptions: BadgeColors[] = ['primary', 'secondary', 'warning', 'caution' ,'danger', 'success'];
```

Comment:
Avoid naming conventions like primary, secondary

I would avoid using terms like "primary," "secondary," etc., as they often lack clear definitions. This can impose unnecessary artificial constraints on something when there is no need for such limitations.

We would eventually run out of ideas for naming things, as evident in this PR itself, which introduces caution and warning. How are these states different?

I would suggest sticking with the actual names from Radix and use a prop like color-scheme which can then became a standard across components.

Reply:
@absurdiya @scmmishra PTAL.